mbofui/app/js/app.js

```diff
@@ -1,8 +1,20 @@
 'use strict';
 /* global angular */
 
-var mbofuiApp = angular.module('mbofuiApp', []);
+var ocellus = angular.module( "ocellus", [ 'ui-leaflet' ] );
```

Comment:
Minor annoyance: Single quotes or double?  I always use single unless I need double for a specific instance.